### PR TITLE
Fix downward api tests to output node allocatable not node capacity

### PR DESCRIPTION
**Pull request by aveshagarwal**
@kubernetes/rh-cluster-infra @derekwaynecarr

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/31535)

### CODE REVIEW

**Comment by aveshagarwal** [comment]
In the PR kubernetes/kubernetes#29639, the default limits have been changed to node allocatables from node capacities.

File: test/e2e/common/downward_api.go [hunk 1]
```diff
@@ -128,7 +128,7 @@ var _ = framework.KubeDescribe("Downward API", func() {
 		testDownwardAPI(f, podName, env, expectations)
 	})
 
-	It("should provide default limits.cpu/memory from node capacity", func() {
+	It("should provide default limits.cpu/memory from node allocatable", func() {
```
Comment (fejta) [inline test/e2e/common/downward_api.go]: Please `update test/test_owners.csv` with this new name.

Comment (aveshagarwal) [inline test/e2e/common/downward_api.go]: @fejta done. thanks.

**Comment by fejta** [comment]
LGTM, thank you!

**Pull request merged**